packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_model.dart

```diff
@@ -116,6 +116,13 @@ class PathError extends CommonError {
   const PathError(title, explanation, fixDetails)
       : super(title, explanation, fixDetails);
 
+  // Activity should have deep link enabled flag.
+  static const missingFlag = PathError(
+    'Activity is missing deep link enabled flag',
```

Comment:
nit: "...missing the flag to enable deep linking" or "...missing the deep linking enabled flag"